docs/blog/2022-07-20-bun-is-the-future.mdx

```diff
@@ -0,0 +1,239 @@
+Having a large plugin ecosystem is kind of critical for a bundler to support different file formats. For instance, .vue files and .scss are still on the roadmap.
+Just about everyone and their grandma uses .scss, so that's a deal-breaker.
+
+## Other features - Web server and sqlite client
+
+Bun also adds some things you'd usually think of as frameworks to the standard library. Personally, I'm less excited about these more library type features. There are a lot of very good options available in Node for http servers.
+
+Bun's webserver looks pretty bare-bones. Express still works fine for most of the community, even if it is a bit behind the times. (They just added promise support this year).
+Bun's server appears to be very similar to a Cloudflare Worker. Maybe Bun's creator will circle back around to more work on the webserver once every other problem in the entire JavaScript world is fixed.
+
+As for the new SQLite adapter, I guess the normal way to do sqlite in Node is a bit confusing. Most people combine the older `sqlite3` package with the `sqlite` wrapper to add promise support,
+but it's not that bad. Bun's solution looks a bit cleaner, and I won't even bother checking if it's faster.
+
+
+## The problem communicating all this value
+
+I'm concerned it's going to be a little hard to communicate all this value to the community. Bun is like a complete ecosystem replacement, and that's a big shift. I hope that people will get it.
+
+Bun is still very new and currently has no complete documentation. A [very long readme](https://github.com/oven-sh/bun#readme) is the main source of truth
+right now. It's not that hard to throw a docusaurus site together (like the one you're reading right now) and generate typedoc from the TypeScript types which already has thorough inline comments, so I imagine this will be coming soon.
+
+## Other alternatives
+### Deno
+You can probably skip this part if you don't know what Deno is or don't care. Personally, I'm way more excited about Bun than Deno.
+
+[Deno](https://deno.land/), written by the original creator of Node, does claim to resolve some outstanding issues.
+It makes es-modules the default, adds first-party TypeScript support (no need to transpile NPM modules before publishing), and so on. In my opinion
+it introduces other serious shortcomings along the way. The elephant in the room is that Deno has made enough breaking changes to package resolution and syntax that **it isn't
+compatible with the existing NPM ecosystem.** This means that Deno needs to be a completely new ecosystem of libraries. There has been a bit of development with some early library support,
+but I'd say that clout can only get you so much buy-in, especially when so much momentum is involved.  There are a few other things in Deno that strike me as half-baked, like the [lack of a package.json](https://deno.land/manual/examples/manage_dependencies),
```

Comment:
It may be worth mentioning you can use [import maps](https://deno.land/manual@v1.15.2/npm_nodejs/import_maps) to alleviate this problem but it's still annoying.